### ISSUE TITLE
targetSDK is missing in JSON output

### ISSUE THREAD

**Issue by fetttobse** [google/play-work#32]
I don't know if it is a bug or a missing feature:

If I import the JSON output from the externallyhosted.py script to Google Play Console, the "Target SDK" field in release management is empty ("-"). 

As from November 1, 2019, Google requires a target API level of 28+ for new apps and/or updates, this will probably cause some problems. As no target SDK is provided in the JSON, Google seems to use the minimum SDK value (which is 24+ in this case) and warns me, that the app won't be compliant with the new rules.

Is it planned to add the targetSDK to the JSON file in the future? 

Thank you very much in advance and best regards!

**Comment by hdurer** [comment]
The target sdk should not be relevant for this feature.  See (issue 26)]https://github.com/google/play-work/issues/26].

When uploading externally hosted apks the target sdk version should not be checked and IIRC the value isn't even checked in the json file.

**Comment by fetttobse** [comment]
Thank you for the responst @hdurer , maybe Google changed this as well and is now checking the target sdk version. I got an email from Google, just a few quotes below:

> Hello Google Play Developer, 
This is a reminder that starting November 1, 2019, updates to apps and games on Google Play will be required to target Android 9 (API level 28) or higher. After this date, the Play Console will prevent you from submitting new APKs with a targetSdkVersion less than 28. 

>Action required 
Please ensure that your apps are configured to target at least Android 9 (API level 28) by November 1, 2019. For technical advice on how to change your app's target API level to meet these requirements, refer to the migration guide. 

They included a list in the email with all my applications that are affected. The list included all three of my externally hosted apps, for which I created the JSON with the externallyhosted.py script.

I will try to open a ticket with Google to make sure that the targetAPK level won't be checked in te future for externally hosted apps, and that I can carry on with my process for new apps. I will update this issue as soon as I get an answer.

Thank you very much for your help so far!

**Comment by fetttobse** [comment]
Hello again, @hdurer 

I finally got a response from Google Support:

> I'm sorry that we can't verify the comment that you've got. 

> But when you receive the alert email from our team, since November 2018, all app updates on Google Play have been required to target API level 26 (Android 8.0) or higher. 

> You can refer to this blog post
[https://android-developers.googleblog.com/2019/02/expanding-target-api-level-requirements.html](https://android-developers.googleblog.com/2019/02/expanding-target-api-level-requirements.html)

So what I understand is the following:
Since November __2018__ apps are required to target API level 26. We uploaded externally hosted apps after November 2018 w/o the targetSDK (because it is not exported to the JSON) and did not get blocked or anything. 

So I assume you are right and the targetSDK is not relevant for externally hosted apps, and this should not change with the raise to API level 28 in November __2019__. I will now close this issue and hope Google won't change it's mind on this.

Thank you very much for your help!
